### PR TITLE
wr-328: Fix the displaying articles on mobile narrow screens

### PR DESCRIPTION
**Pull request by kovAnya**

### CODE REVIEW

File: frontend/src/libs/components/tags/tags.tsx [hunk 1]
```diff
@@ -12,7 +12,13 @@ type Properties = {
 };
 
 const Tags: React.FC<Properties> = ({ className, tags }) => (
-  <ul className={getValidClassNames(styles.tags, className)}>
+  <ul
+    className={getValidClassNames(
+      styles.tags,
+      className,
+      tags.length === 1 ? styles.singleTag : '',
```
Comment (andanf-e) [inline frontend/src/libs/components/tags/tags.tsx]: And **B31**
```suggestion
      tags.length === 1 ?&& styles.singleTag,
```

File: frontend/src/libs/components/tags/tags.tsx [hunk 2]
```diff
@@ -12,7 +12,13 @@ type Properties = {
 };
 
 const Tags: React.FC<Properties> = ({ className, tags }) => (
-  <ul className={getValidClassNames(styles.tags, className)}>
+  <ul
+    className={getValidClassNames(
+      styles.tags,
+      className,
+      tags.length === 1 && styles.singleTag,
```
Comment (andanf-e) [inline frontend/src/libs/components/tags/tags.tsx]: B31

Comment (kovAnya) [reply frontend/src/libs/components/tags/tags.tsx]: can I name constant SINGLE_NUMBER? Have no ideas how to name better

File: frontend/src/libs/components/tags/libs/constants/single-number.constant.ts [hunk 3]
```diff
@@ -0,0 +1,3 @@
+const SINGLE_NUMBER = 1;
```
Comment (andanf-e) [inline frontend/src/libs/components/tags/libs/constants/single-number.constant.ts]: SINGLE_ELEMENT_ARRAY_LENGTH may be more proper name

**Review by andanf-e — APPROVED** [review]

**Pull request merged**